pkg/printers/internalversion/describe.go

```diff
@@ -1137,8 +1137,13 @@ func describePersistentVolume(pv *api.PersistentVolume, events *api.EventList) (
 		w.Write(LEVEL_0, "Name:\t%s\n", pv.Name)
 		printLabelsMultiline(w, "Labels", pv.Labels)
 		printAnnotationsMultiline(w, "Annotations", pv.Annotations)
+		w.Write(LEVEL_0, "Finalizers:\t%v\n", pv.ObjectMeta.Finalizers)
 		w.Write(LEVEL_0, "StorageClass:\t%s\n", helper.GetPersistentVolumeClass(pv))
-		w.Write(LEVEL_0, "Status:\t%s\n", pv.Status.Phase)
+		if pv.ObjectMeta.DeletionTimestamp != nil {
+			w.Write(LEVEL_0, "Status:\tTerminating (since %s)\n", pv.ObjectMeta.DeletionTimestamp.Time.Format(time.RFC1123Z))
```

Comment:
Why not use `translateTimestamp` like objectmeta printing normally does?

Reply:
It follows pattern introduced in Pod: https://github.com/NickrenREN/kubernetes/blob/b3f370201d1f85416ecf03c34d51ab7d51ef4599/pkg/printers/internalversion/describe.go#L638

Reply:
I think that was just an oversight.  Either when it first came in or when the translateTimestamp was made.  DeletionTimestamp should parallel CreationTimestamp.

Reply:
ok, @NickrenREN, can you please change all `DeletionTimestamps` to `translateTimestamp` ?

Reply:
yeah, sure, will do